daemon/api_snaps.go

```diff
@@ -196,21 +196,22 @@ type snapInstruction struct {
 	Action string `json:"action"`
 	Amend  bool   `json:"amend"`
 	snapRevisionOptions
-	DevMode                bool                   `json:"devmode"`
-	JailMode               bool                   `json:"jailmode"`
-	Classic                bool                   `json:"classic"`
-	IgnoreValidation       bool                   `json:"ignore-validation"`
-	IgnoreRunning          bool                   `json:"ignore-running"`
-	Unaliased              bool                   `json:"unaliased"`
-	Purge                  bool                   `json:"purge,omitempty"`
-	SystemRestartImmediate bool                   `json:"system-restart-immediate"`
-	Transaction            client.TransactionType `json:"transaction"`
-	Snaps                  []string               `json:"snaps"`
-	Users                  []string               `json:"users"`
-	ValidationSets         []string               `json:"validation-sets"`
-	QuotaGroupName         string                 `json:"quota-group"`
-	Time                   string                 `json:"time"`
-	HoldLevel              string                 `json:"hold-level"`
+	DevMode                bool                            `json:"devmode"`
+	JailMode               bool                            `json:"jailmode"`
+	Classic                bool                            `json:"classic"`
+	IgnoreValidation       bool                            `json:"ignore-validation"`
+	IgnoreRunning          bool                            `json:"ignore-running"`
+	Unaliased              bool                            `json:"unaliased"`
+	Purge                  bool                            `json:"purge,omitempty"`
+	SystemRestartImmediate bool                            `json:"system-restart-immediate"`
+	Transaction            client.TransactionType          `json:"transaction"`
+	Snaps                  []string                        `json:"snaps"`
+	Users                  []string                        `json:"users"`
+	SnapshotOptions        map[string]snap.SnapshotOptions `json:"options"`
```

Comment:
This should be `snapshot-options` since the snapInstruction is not specific to snapshots

Reply:
Thanks, I understand what you are saying. This will require a spec change. Will check with @mvo5 / @pedronis

Reply:
Sorry, I forgot to add context. This is related to the comments I made on the spec document about it and the PR not being consistent

Reply:
`Options` was renamed to `SnapshotOptions` and the tag to `snapshot-options` as agreed in the spec meeting on 27 Feb.